rpcs3/Emu/Cell/Modules/sceNp.h

```diff
@@ -853,17 +853,15 @@ struct SceNpDrmOpenArg
 // NP communication ID structure
 struct SceNpCommunicationId
 {
-	char data[9];
-	char term;
+	char data[9 + 1]; // char term;
```

Comment:
I'm sure it's better like this, but aren't these structs copied directly from official documentation/headers? Feels weird to change it

Reply:
It shouldn't feel weird to make minor sensible changes. Plus they aren't directly copied or something, they are just compatible with those in PS3.